Author: Roland Rosenfeld <roland@debian.org>
Description: For fonts >=42pt use the given font size in TeX instead of
 restricting this to 42pt. This allows to create posters using
 \usepackage{type1cm}. See http://linuxgazette.vlsm.org/issue96/artime.html
 and Bug#343139
Bug-Debian: http://bugs.debian.org/343139

--- a/LATEX.AND.XFIG
+++ b/LATEX.AND.XFIG
@@ -55,7 +55,7 @@
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% LaTeX Preamble %%%%%%%%%%%%%%%%%%%%%%%%%
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
  
-\documentstyle[12pt,bezier,amstex]{article}  % include bezier curves
+\documentclass[12pt,bezier,amstex]{article}  % include bezier curves
 \renewcommand\baselinestretch{1.0}           % single space
 \pagestyle{empty}                            % no headers and page numbers
 \oddsidemargin -10 true pt      % Left margin on odd-numbered pages.
@@ -72,7 +72,9 @@
 \usepackage{epsfig}
 
 \usepackage{graphicx}           % Graphics package
- 
+
+\usepackage{type1cm}            % Necessary for fonts bigger than 42pt
+
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Document Beginning %%%%%%%%%%%%%%%%%%%%%
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
--- a/fig2dev/dev/texfonts.h
+++ b/fig2dev/dev/texfonts.h
@@ -83,9 +83,11 @@
 #endif
 #define TEXFONT(F)	(texfontnames[((F) <= MAX_FONT) ? (F) : (MAX_FONT-1)])
 
-
+/*
 #define TEXFONTSIZE(S)	(texfontsizes[((S) <= MAXFONTSIZE) ? (int)(round(S))\
 				      				: (MAXFONTSIZE-1)])
+*/
+#define TEXFONTSIZE(S)	(((S) <= MAXFONTSIZE) ? texfontsizes[(int)(round(S))] : (S))
 #define TEXFONTMAG(T)	TEXFONTSIZE(T->size*(rigid_text(T) ? 1.0 : fontmag))
 
 void setfigfont( F_text *text );		/* genepic.c */
